build-tools/rpm/macros.yast

```diff
@@ -79,7 +79,7 @@ fi \
     %if %{with yast_run_ci_tests} \
       if [ -f "configure.in.in" -o -f "configure.ac.in" ]; then \
           # TODO: fix the check:ci task to also work with autotools based modules \
-          %{__make} check \\\
+          %{__make} check %{?jobs:-j%jobs}%{?!jobs:%{?_smp_mflags:%_smp_mflags}} \\\
```

Comment:
`%{?SYMBOL:SUBST}` if SYMBOL is defined, use SUBST
`%{?!SYMBOL:SUBST}` if SYMBOL is **not** defined, use SUBST

Curiously, there does not seem to be an official reference for this